app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,18 @@
+class RegistrationsController < Devise::RegistrationsController
+
+  skip_before_action :require_no_authentication, only: [:new, :create]
```

Comment:
From Tom in slack: "this line of code is necessary to allow admins to create an additional admin account while still logged in as admins. If this code isn't there, it'll throw an error if I try to access the admins/sign_up route while currently logged in"

Reply:
Added a comment in the code to explain this